apps/dotcom/client/src/tla/components/TlaEditor/TlaEditorTopLeftPanel.tsx

```diff
@@ -94,7 +96,10 @@ export function TlaEditorTopLeftPanelSignedIn() {
 		[app, fileId]
 	)
 
-	const handleRenameAction = () => setIsRenaming(true)
+	const handleRenameAction = () => {
+		setIsRenaming(true)
+		trackEvent('rename-file', { source: 'file-menu' })
```

Comment:
This is not the actual rename, it's just changing the local state. We might need to move this to `handleFileNameChange` or maybe even `TlaFileNameEditor`?

Reply:
I thought about putting it in handleFileNameChange, but assumed we wouldn't want to track every keystroke. I'll put it in handleEditingComplete within TlaFileNameEditor.